integration-patterns.md

```diff
@@ -69,3 +69,27 @@ have to include
 <script src="{% new_static 'hqwebapp/js/toggles.js' %}"></script>
 <script src="{% new_static 'style/js/bootstrap3/main.js' %}"></script>
 ```
+
+## Remote Method Invocation
```

Comment:
This could use a quick blurb explaining what RMI is and why it's useful.  The first line touches on it, but an explicit "Remote Method Invocation is X and it's useful because X" would be nice.